src/main/java/ch/wisv/chue/states/HueState.java

```diff
@@ -0,0 +1,13 @@
+package ch.wisv.chue.states;
+
+import ch.wisv.chue.HueCommand;
+
+/**
+ * @author Sander Ploegsma
+ */
+public abstract class HueState extends HueCommand {
+
+    public HueState() {
+
+    }
```

Comment:
Mja daar ben ik het mee eens. Ik had eerst die twee gemaakt tot ik erachter kwam dat alle code die erin stond hetzelfde was (staat nu in `HueCommand`). Ben er alleen wel fan van dat je dus geen events als states kan laden en zo, dus als er iemand een andere manier weet om dat te doen? :smiley: